plog/testlogger.go

```diff
@@ -11,6 +11,11 @@ func TestLogger(t *testing.T) Logger {
 		t.Helper()
 
 		j, _ := json.MarshalIndent(e, "", "  ")
-		t.Logf("%s", j)
+
+		if e.Severity == SevError {
+			t.Errorf("test produced a log entry with error severity: %s", j)
```

Comment:
optional:
Why is the `"test produced a log entry with error severity: ` logged for error messages?

Reply:
Is logging with `t.Error()`, to cause the test to fail.